doc/running.rst

```diff
@@ -55,6 +55,9 @@ The following default values are used, corresponding to the command line default
    * - codegen_opts
      - Optional[Mapping[str, Any]]
      - (Optional) A JSON equivalent Python dictionary containing additional options for the target platform code generator. These options are specific to a given target platform, see for example :ref:`Running NESTML with custom templates`.
+   * - redirect_cmake_output
```

Comment:
I think it is better to have this in `codegen_opts` as this is specific to NEST target